### ISSUE TITLE
Settings Panel settings does not have proper linkage between labels and fields

### ISSUE THREAD

**Issue by hartra344** [Azure/LogicAppsUX#1967]
### Describe the Bug with repro steps

use screen reader

### Browser

Chrome
